README.md

```diff
@@ -1,2 +1,23 @@
-# community
-A place for community-wide issues, discussion, resources, files and sharing
+# Welcome to the Libero Community
+This is a place for community-wide issues, discussion, resources, files and sharing.
+## What is Libero?
+Libero is eLife’s open-source platform of services and applications to help publishers do more with everything they publish.
+
+Built in a modular fashion from the ground up, Libero adjusts to each journal’s individual needs to help deliver scholarly content to the highest standards in design and usability, while also providing machine-readable versions and lowering publishing costs.
+
+It is a community-driven effort with decision making, coding and deployment all done in the open.
+
+More information at https://libero.pub and https://elifesciences.org/about
+
+## Requests for Comments
+In the interests of community involvement and openness, any member of the community can issue a request for comments (RFC) for any major decision that affects more than one party in the community or makes a significant difference to a component, module, application or service in the platform.
+
+RFCs should be posted as issues in this repository, or in the repository most relevant to the RFC. You can [find open RFCs for the whole community by searching this repository's issues for RFC](https://github.com/libero/community/issues?q=is%3Aissue+is%3Aopen+rfc)
+
+There is a time limit on RFCs and when a decision has been made following a discussion (on the issue or in person), the issue will be updated and closed. Where appropriate a corresponding architectural decision record may be created.
```

Comment:
link to https://www.thoughtworks.com/radar/techniques/lightweight-architecture-decision-records ?